website/static/js/pages/component-settings-page.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var $ = require('jquery');
+var bootbox = require('bootbox');
+var Raven = require('raven-js');
+var $osf = require('js/osfHelpers.js');
+var getConfirmationCode = require('js/projectSettings.js').getConfirmationCode;
```

Comment:
I'm not sure it is general enough to be added to helpers. A separate file similar to projectSettings.js or the helpers would probably work fine.